homeassistant/components/sensor/zwave.py

```diff
@@ -33,26 +34,36 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     # if 1 in groups and (NETWORK.controller.node_id not in
     #                     groups[1].associations):
     #     node.groups[1].add_association(NETWORK.controller.node_id)
+    name = '{}.{}'.format(DOMAIN, zwave.object_id(value))
+    node_config = customize.get_overrides(hass, zwave.DOMAIN, name)
```

Comment:
Force update only does something if both the state and the attributes are the same. So probably makes sense just for sensors?

What would be a use case of a switch to be forced update ?